conf/layer.conf

```diff
@@ -9,3 +9,4 @@ BBFILE_COLLECTIONS += "libaktualizr-demo"
 BBFILE_PATTERN_libaktualizr-demo = "^${LAYERDIR}/"
 BBFILE_PRIORITY_libaktualizr-demo = "7"
 
+LAYERSERIES_COMPAT_libaktualizr-demo = "warrior"
```

Comment:
Should this also include zeus and dunfell as well, if it is compatible with them? Otherwise this seems like it might belong in a warrior branch.

Reply:
It is probably compatible, but we've never tested it. And we have only one branch here.

Reply:
I think something like this should be documented or made more obvious. To me this is unexpected. It's fine for the moment if it solves immediate problems. I think it's reasonable to expect that it should also be compatible with newer branches, though.